enforce_codeowners/keras_bot/importTester.py

```diff
@@ -0,0 +1,122 @@
+        lineno+=1
+        #Ingore lines withing multi line comments
+        if '\"\"\"' in line:
+            comment = not comment
+        #Empty line or comment line
+        if line == "" or comment == True or '#' in line:
+            pass
+        else:
+            split_line = line.split()
+            #Import
+            if split_line[0] == 'import':
+                module_split = split_line[1].split('.')
+                #Check if module is an ignored library
+                if module_split[0] in ignore:
+                    pass 
+
+            #ImportFrom
+            elif split_line[0] == 'from' and len(split_line) >= 3:
+                #Check if module is an ignored library or line doesnt contain import
+                if split_line[1] in ignore or split_line[2] != 'import':
+                    pass
+                #Check if import is absolute or relative
+                elif split_line[1].startswith('.'):
+                    string = str(file) + " contains a relative import on line " + str(lineno)
+                    errs.append(string)
+                else:
+                    module_split = split_line[1].split('.')
+                    if module_split[0] in ignore:
+                        continue
+
+    return errs
+
+def check_relative(file):
+    
+    with open(file) as f:
+        imports = f.read().split("\n")
+
+    errs = []
+    lineno=0
+
+    comment = False
+
+    for line in imports:
+        lineno+=1
+        #Ingore lines withing multi line comments
+        if '\"\"\"' in line:
+            comment = not comment
+        #Empty line or comment line
+        if line == "" or comment == True or '#' in line:
+            pass
+        else:
+            split_line = line.split()
+            #Import
+            if split_line[0] == 'import':
+                module_split = split_line[1].split('.')
+                #Check if module is an ignored library
+                if module_split[0] in ignore:
+                    continue
+                else:
+                    string = str(file) + " contains an absolute import on line " + str(lineno)
+                    errs.append(string)
+
+            #ImportFrom
+            elif split_line[0] == 'from' and len(split_line) >= 3:
+                #Check if module is an ignored library or line doesnt contain import
+                if split_line[1] in ignore or split_line[2] != 'import':
+                    pass
+                #Check if import is absolute or relative
+                elif split_line[1].startswith('.'):
+                    pass
+                else:
+                    module_split = split_line[1].split('.')
+                    if module_split[0] in ignore:
+                        continue
+                    else:
+                        string = str(file) + " contains a relative import on line " + str(lineno)
+                        errs.append(string)
+
+    return errs
+
+def checkImports(dirPath, style):
```

Comment:
This function name does not follow PEP8. See https://www.python.org/dev/peps/pep-0008/